app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :email, presence: true
```

Comment:
- [ ] **[OPTIONAL]** We can validate all attributes in one liner if they all share the same validator
`validates :username, :email, :password, presence: true`